### ISSUE TITLE
Dropbox login not working

### ISSUE THREAD

**Issue by paologas91** [Kyrodan/KeeAnywhere#132]
Im using 1.4.1 with keepass 2.38 portable version. When trying to login to Dropbox, i can't click on the button to access after inserting email and password (As you can see in the photo).
![screenshot 1](https://user-images.githubusercontent.com/27777225/39443818-0718c704-4cb6-11e8-8b06-4cdce8e511ac.png)

**Comment by JonasBonno** [comment]
The problems seems to relate to the new Internet Explorer build in Windows 10 build 17134. Opening the authorize site in normal IE11 gives the same issue. 

Running the site in developer mode with IE9 emulation fixes it, but I was not able to set IE9 mode for the plugin. Tried playing with IE FEATURE_BROWSER_EMULATION reg. options but with no luck.
The code for IE modes are coded into the plugin and will changes to 11001 on KeePass start.
I tried changing the IE build in reg number faking a IE9 version, so the plugin set the code to 9000 but it did not help either.

Sources: 
- https://www.dropboxforum.com/t5/API-Support-Feedback/Sign-In-button-Greyed-OUT/m-p/255388/highlight/true#M14780 
- https://docs.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/general-info/ee330730(v=vs.85)#browser_emulation 
- https://github.com/Kyrodan/KeeAnywhere/blob/master/KeeAnywhere/BrowserHelper.cs
